orc8r/cloud/go/services/metricsd/servicers/servicer.go

```diff
@@ -63,6 +63,14 @@ func (srv *MetricsControllerServer) Collect(ctx context.Context, in *protos.Metr
 	}
 
 	hardwareID := in.GetGatewayId()
+        checkID, err := protos.GetGatewayIdentity(ctx)
```

Comment:
check your whitespace formatting. `go fmt` should set you straight

Reply:
All source files formatted using go fmt.